gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -434,6 +434,97 @@ gst_tensor_filter_finalize (GObject * object)
   G_OBJECT_CLASS (parent_class)->finalize (object);
 }
 
+/**
+ * @brief Parse the string of model
+ * @param info tensors info structure
+ * @param model_file the prediction model path
+ * @param model_file_sub the initialize model path
+ * @return number of parsed model path
+ */
+static guint
+gst_tensors_parse_modelpaths_string (GstTensorFilterProperties * prop,
+    const gchar * model_files)
+{
+  gchar **models;
+  gchar **model_0;
+  gchar **model_1;
+  guint num_models = 0;
+  guint num_model_0 = 0;
+  guint num_model_1 = 0;
+
+  g_return_val_if_fail (prop != NULL, 0);
+
+  if (model_files) {
+
+    models = g_strsplit_set (model_files, ",", -1);
+
+    num_models = g_strv_length (models);
+
+    if (num_models == 1) {
+      prop->model_file = g_strdup (models[0]);
+      return num_models;
```

Comment:
free data g_strfreev (models) before return

Reply:
it's applied